site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -361,18 +358,17 @@
         "\n",
         "Some of the columns in the CSV data are categorical columns. That is, the content should be one of a limited set of options.\n",
         "\n",
-        "In the CSV, these options are represented as text. This text needs to be converted to numbers before the model can be trained. To facilitate that, we need to create a list of categorical columns, along with a list of the options available in each column."
+        "Use the `tf.feature_column` API to create a collection with a `tf.feature_column.indicator_column` for each categorical column.\n",
+        "\n"
       ]
     },
     {
       "cell_type": "code",
-      "execution_count": 0,
       "metadata": {
```

Comment:
In the [Colab notebook preview](https://colab.sandbox.google.com/github/adammichaelwood/tf-docs/blob/csv-feature-columns/site/en/r2/tutorials/load_data/csv.ipynb), some of the data cells are hidden---is that intended?

Reply:
No. And I don't know how to stop it from happening.